app/pages/application-details.tsx

```diff
@@ -2,14 +2,17 @@ import React, {Component} from 'react';
 import {graphql} from 'react-relay';
 import {applicationDetailsQueryResponse} from 'applicationDetailsQuery.graphql';
 import {NextRouter} from 'next/router';
+import {Row, Col} from 'react-bootstrap';
 import IncentiveCalculatorContainer from '../containers/Incentives/IncentiveCalculatorContainer';
 import ApplicationStatusContainer from '../containers/Applications/ApplicationStatusContainer';
 import DefaultLayout from '../layouts/default-layout';
 import ApplicationDetail from '../containers/Applications/ApplicationDetailsContainer';
+import ApplicationComments from '../containers/Applications/ApplicationCommentsContainer';
```

Comment:
```suggestion
import ApplicationComments from 'containers/Applications/ApplicationCommentsContainer';
```

This works now. No need to write `..`, you can write relative imports starting from the root of the app